app/script/conversation/ConversationMapper.js

```diff
@@ -157,123 +157,123 @@ z.conversation.ConversationMapper = class ConversationMapper {
+        conversationEntity.setTimestamp(last_server_timestamp, z.entity.Conversation.TIMESTAMP_TYPE.LAST_SERVER);
       }
 
       if (muted_timestamp) {
-        conversation_et.setTimestamp(muted_timestamp, z.entity.Conversation.TIMESTAMP_TYPE.MUTED);
-        conversation_et.muted_state(self_state.muted_state);
+        conversationEntity.setTimestamp(muted_timestamp, z.entity.Conversation.TIMESTAMP_TYPE.MUTED);
+        conversationEntity.muted_state(selfState.muted_state);
       }
 
       if (status !== undefined) {
-        conversation_et.status(status);
+        conversationEntity.status(status);
       }
 
       if (verification_state !== undefined) {
-        conversation_et.verification_state(verification_state);
+        conversationEntity.verification_state(verification_state);
       }
 
       // Backend states
-      const {otr_archived, otr_muted} = self_state;
+      const {otr_archived, otr_muted} = selfState;
 
       if (otr_archived !== undefined) {
-        const otr_archived_timestamp = new Date(self_state.otr_archived_ref).getTime();
-        conversation_et.setTimestamp(otr_archived_timestamp, z.entity.Conversation.TIMESTAMP_TYPE.ARCHIVED);
-        conversation_et.archived_state(otr_archived);
+        const otrArchivedTimestamp = new Date(selfState.otr_archived_ref).getTime();
+        conversationEntity.setTimestamp(otrArchivedTimestamp, z.entity.Conversation.TIMESTAMP_TYPE.ARCHIVED);
+        conversationEntity.archived_state(otr_archived);
       }
 
       if (otr_muted !== undefined) {
-        const otr_muted_timestamp = new Date(self_state.otr_muted_ref).getTime();
-        conversation_et.setTimestamp(otr_muted_timestamp, z.entity.Conversation.TIMESTAMP_TYPE.MUTED);
-        conversation_et.muted_state(otr_muted);
+        const otrMutedTimestamp = new Date(selfState.otr_muted_ref).getTime();
+        conversationEntity.setTimestamp(otrMutedTimestamp, z.entity.Conversation.TIMESTAMP_TYPE.MUTED);
+        conversationEntity.muted_state(otr_muted);
       }
 
       if (disablePersistence) {
-        conversation_et.setStateChangePersistence(true);
+        conversationEntity.setStateChangePersistence(true);
       }
 
-      return conversation_et;
+      return conversationEntity;
     }
   }
 
   /**
    * Creates a conversation entity from backend JSON data.
    *
    * @private
-   * @param {Object} conversation_data - Either locally stored or backend data
-   * @param {number} [initial_timestamp] - Initial timestamp for conversation in milliseconds
+   * @param {Object} conversationData - Either locally stored or backend data
+   * @param {number} [initialTimestamp] - Initial timestamp for conversation in milliseconds
    * @returns {Conversation} Mapped conversation entity
    */
-  _create_conversation_et(conversation_data, initial_timestamp) {
-    if (conversation_data === undefined || !Object.keys(conversation_data).length) {
+  _createConversation(conversationData, initialTimestamp) {
```

Comment:
Why not `_createConversationEntity()`? The variable inside the function is also called `conversationEntity`.

Reply:
Because I was lazy